lms/djangoapps/courseware/tests/test_transformers.py

```diff
@@ -0,0 +1,158 @@
+"""
+Test the behavior of the GradesTransformer
+"""
+
+import datetime
+import pytz
+
+from student.tests.factories import UserFactory
+from lms.djangoapps.course_blocks.api import get_course_blocks
+from lms.djangoapps.course_blocks.transformers.tests.helpers import CourseStructureTestCase
+from ..transformers import GradesBlockTransformer
+
+
+class GradesBlockTransformerTestCase(CourseStructureTestCase):
+    """
+    Verify behavior of the GradesBlockTransformer
+    """
+
+    TRANSFORMER_CLASS_TO_TEST = GradesBlockTransformer
+
+    problem_metadata = {
+        u'graded': True,
+        u'weight': 1,
+        u'due': datetime.datetime(2099, 03, 15, 12, 30, 00, tzinfo=pytz.utc),
```

Comment:
I realize the octal notation doesn't really have any effect here, but please don't use it unless you really need to.